lib/core/src/lib/notifications/components/notification-history.component.ts

```diff
@@ -53,17 +53,19 @@ export class NotificationHistoryComponent implements OnDestroy, OnInit, AfterVie
     onDestroy$ = new Subject<boolean>();
     notifications: NotificationModel[] = [];
     paginatedNotifications: NotificationModel[] = [];
+    unreadNotifications: NotificationModel[] = [];
     pagination: PaginationModel;
+    badgeHidden: boolean;
 
-    constructor(
-        private notificationService: NotificationService,
-        public storageService: StorageService,
-        public cd: ChangeDetectorRef) {
-
-    }
+    constructor(private notificationService: NotificationService, public storageService: StorageService, public cd: ChangeDetectorRef) {}
 
     ngOnInit() {
         this.notifications = JSON.parse(this.storageService.getItem(NotificationHistoryComponent.NOTIFICATION_STORAGE)) || [];
+        this.unreadNotifications =
+            JSON.parse(this.storageService.getItem(NotificationHistoryComponent.NOTIFICATION_STORAGE))?.filter(
+                (notification) => !notification.read
```

Comment:
provide the notification type.
notification: NotificationModel

Reply:
added type